plugins/livelog/config-schema.yml

```diff
@@ -0,0 +1,14 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              Config
+description: |-
+  LiveLog config
+type:               object
+properties:
+  liveLogExe:
+    title:          LiveLogExecutable
+    description: |-
+      The executable (.exe file) to run the livelog service
+    type:           string
+additionalProperties: false
+required:
+  - liveLogExe
```

Comment:
@jonasfj Note - this file is for illustration - I *DON'T* propose we use an .exe file for the livelog service - just stole the config from generic worker for illustration.